### DISCUSSION TITLE
Support for Liqo

### DISCUSSION THREAD

**Discussion by robotica72** [liqotech/liqo#1841] [Q&A]
Hi -

Might be a strange question, but is there a vendor that offers support for running Liqo in a production environment?   

I can use it for some workloads, but I will have a number of other workloads running Tier 0 applications that  require vendor support in the case of a big emergency.

Thanks!

**Comment by frisso** [comment]
Yes, we have, although this is not yet 100% public.
We can sync offline for the details.

**Comment by robotica72** [comment]
Thanks @frisso - Catching up offline would be great.  My ask is unoffiicial as well :)  But I need to get all of my ducks in a row before I can present this to my Exco team for consideration.  Do you have a preferred e-mail I can send a contact to?

**Reply by frisso** [reply]
Just follow my GitHub profile. Thanks!

**Comment by robotica72** [comment]
Perfect!  Thanks
